packages/features/ee/teams/components/TeamAvailabilityModal.tsx

```diff
@@ -88,7 +88,7 @@ export default function TeamAvailabilityModal(props: Props) {
           <div className="col-span-1 max-h-[500px]">
             {props.team && props.member && (
               <TeamAvailabilityTimes
-                teamId={props.team.id}
+                teamId={props.team.id ?? 0}
```

Comment:
Why are we setting a teamId of 0? What is the case where it happens

Reply:
props.team could be undefined. I'll add a props.team check at top level.